backend/schemas/source/sections/AuditInformation.schema.jsonnet

```diff
@@ -110,6 +110,22 @@ local AuditInformation = Types.object {
   title: 'AuditInformation',
 };
 
+// WARNING: If a comment is the last thing in a Jsonnet file, then the formatter will
+// ALWAYS add a new line to the end of the file, meaning this file will change every
+// time we run the formatter.
+//
+// The directions below suggest that the array should be added to the end of the file
+// to run local tests. There might be a better way: create a new file that we run the tests
+// against that imports from this one, and wraps the AuditInformation object in an array.
+// This will have the same effect, but make it testable without modifying this file.
+
+//To manually text against data_fixtures/audit/excel_schema_test_files/audit-information-pass-01.js
```

Comment:
This comment was meant to be a hint for whoever wants to do a quick manual testing against an array of AuditInformation.  If it causes issue then it is better to get rid of it. Moving up does not make sense as the item 1 below is now out of context.